src/python/pants/build_graph/address.py

```diff
@@ -244,18 +244,20 @@ class BuildFileAddress(Address):
   :API: public
   """
 
-  def __init__(self, build_file, target_name=None):
+  def __init__(self, build_file=None, rel_path=None, target_name=None):
```

Comment:
thanks - I had fixed all the usages within the pants codebase to be named arg invocation already. makes sense to move to the end too.